packages/oui-criteria-container/src/criteria-container.component.js

```diff
@@ -0,0 +1,11 @@
+import controller from "./criteria-container.controller";
+
+export default {
+    template: "<ng-transclude></ng-transclude>",
+    transclude: true,
+    controller,
+    bindings: {
+        criteria: "=model",
+        onChange: "&?"
```

Comment:
You should remove "?" else you'll be forced to check if (this.onChange) before triggering on
Change.
To reproduce the bug, remove the callback for on-change and trigger onChange event. An exception will raise.